Classes/Domain/Repository/FaqRepository.php

```diff
@@ -1,6 +1,7 @@
 <?php
 namespace Tev\TevFaqs\Domain\Repository;
 
```

Comment:
Some comment as above - none of this code is needed. You just need to remove all of the `$querySettings` modifications completely from `initializeObject()`